src/secrets.js

```diff
@@ -14,6 +14,12 @@ export const MIXPANEL_PROJECT_TOKEN =
 export const WS_RELAY_URL = process.env.REACT_APP_WS_RELAY_URL;
 export const IS_BURN = process.env.REACT_APP_IS_BURN;
 
+// @debt doing this for GitHub demo
+export const AGORA_APP_ID = "6f46c1a52afc445c94901fa64877cd05";
+export const AGORA_CHANNEL = "TestSparkle";
+export const AGORA_TOKEN =
+  "0066f46c1a52afc445c94901fa64877cd05IAAzOVWNGWxPaIM6/imvMqJoOMLMl6sn4k9OSmlaL7PZ0bjtc7kAAAAAEAAm+nFWKabUYAEAAQApptRg";
```

Comment:
This should NEVER have been done, not even for demo purposes. This is 1000% security issue. Please NEVER do ANYTHING like this again under ANY circumstances EVER.

Reply:
@mike-lvov Where did these details come from? And have you revoked the token/etc yet? I don't see anything configured in our main account, which makes me think you did this on a private account you have?

Reply:
**Noting for future:** This change was made to triage a quick turnaround needed for a demo. The proper implementation is coming 'soon', and will involve securely and dynamically creating the tokens on the backend, as per best practices.

Reply:
See also: https://github.com/sparkletown/sparkle/pull/1661#issuecomment-868934473